glide.lock

```diff
@@ -1,5 +1,5 @@
-hash: ccd5caad2b8fa46f76338964fe06f5d6a35b9ed36dd5df8da0748e3e3ab99e6a
-updated: 2018-02-19T17:31:51.88595122-08:00
+hash: 997ebe999814737fa4909759114abff8ec65078f19590466b7df0c6ac1060bbc
+updated: 2018-02-19T18:40:27.480656018-08:00
```

Comment:
glide file seems like it changed a lot of files, was this intentional? were these all changes stemming from vendoring the latest btcwallet?

Reply:
The files were changed as we have some "fuzzy" versions set. So we'll like automatically accept a patched version or w/e. In the future we might want to consider turning that off, and always having to *manually* update the glide files for new versions.